### DISCUSSION TITLE
Named terminals display 'undefined' as first line

### DISCUSSION THREAD

**Discussion by Ariemeth** [postmanlabs/postman-flows#381] [Defect]
I've noticed that any output displayed in a named terminal other then default has the term `undefined` displayed before the date and data that was sent to the terminal
![image](https://user-images.githubusercontent.com/541023/186017802-acff6185-f038-42ba-bbb1-3fe76ee68554.png)

same output when sent to the default terminal
![image](https://user-images.githubusercontent.com/541023/186017920-39d24e91-5f57-4c0c-bee7-a187e6cf45f3.png)

**Comment by shrey-bhagat** [comment]
hey @Ariemeth thanks for reporting this bug. We'll fix this soon
